### ISSUE TITLE
Custom data

### ISSUE THREAD

**Issue by SubhadityaMukherjee** [FluxML/model-zoo#254]
Can we please have a tutorial which involves something other than MNIST and CIFAR? 
Especially if it has a custom image dataset. I think it would be a very useful tutorial and would probably get a lot more people interested. 
I was thinking of writing one but I can’t seem to figure out how to yet ):

**Comment by CarloLucibello** [comment]
yes, we will do it as soon as we have the infrastructure in place.  Meanwhile, https://github.com/lorenzoh/DLDatasets.jl could be of help

**Comment by CarloLucibello** [comment]
and also https://github.com/lorenzoh/DataLoaders.jl
